packages/angular/src/errorhandler.ts

```diff
@@ -0,0 +1,118 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { ErrorHandler as AngularErrorHandler, Injectable } from '@angular/core';
+import * as Sentry from '@sentry/browser';
+
+/**
+ * Options used to configure the bahavior of ErrorHandler.
+ */
+export interface ErrorHandlerOptions {
+  logErrors?: boolean;
+  showDialog?: boolean;
+  dialogOptions?: Sentry.ReportDialogOptions;
+  /**
+   * Custom implementation of error extraction from the raw value captured by the Angular.
+   * @param error Value captured by Angular's ErrorHandler provider
+   * @param defaultExtractor Default implementation that can be used as the fallback in case of custom implementation
+   */
+  extractor?(error: unknown, defaultExtractor: (error: unknown) => unknown): unknown;
```

Comment:
Is `unknown` really the best type? We can't make it stricter?

Reply:
I tried to find something more appropriate here, but looking at our issues stream on github, as well as angular's code, it can be really anything here...